images/prow-tests/Dockerfile

```diff
@@ -49,7 +52,7 @@ RUN bazel version > /bazel_version
 RUN bazel shutdown
 
 # Extract ko version
-RUN ko version > /ko_version
+RUN git ls-remote https://github.com/google/ko HEAD | cut -f1 > /ko_version
```

Comment:
This is also wrong, please rebase your PR: remove this line and keep the `RUN ko version` above.

Reply:
fixed, thanks